tethne/serialize/paper.py

```diff
@@ -0,0 +1,178 @@
+"""
+This class will serialize the Tethne Paper to persist in the database;
+
+.. code-block:: python
+>>> from tethne.serialize import paper
+
+"""
+
+
+import re
+import os
+import sys
+from tethne.readers import Corpus, Paper
+from tethne.dao import tethnedao
+import json
+import datetime
+from time import gmtime, strftime
+from uuid import uuid4
+
+import sys
+PYTHON_3 = sys.version_info[0] == 3
+if PYTHON_3:
+    unicode = str
+
+class Serialize:
+
+    paper_source_map = {3: 'wosid', 2: 'url'}
+
+    def __init__(self, corpus, source):
+        self.corpus = corpus
+        self.source = source
+        print tethnedao.getMaxCorpusID()
+        self.corpus_id = tethnedao.getMaxCorpusID()+1
+        self.paperIdMap = {}
+        self.authorIdMap = {}
+        self.authorInstanceIdMap = {}
+
+    def serializeCorpus(self):
+        if self.corpus is not None:
+            corpus_details = [{
+                "model": "django-tethne.corpus",
+                "pk": self.corpus_id,
+                "fields": {
+                "source": self.source,
+                "date_created":strftime("%Y-%m-%d %H:%M:%S", gmtime()),
+                "length" : len(self.corpus),
+                }
+            }]
+        return corpus_details
+
+    def serializePaper(self):
+        pid = tethnedao.getMaxPaperID();
+        for paper in self.corpus:
+            pid = pid+1
+            if not hasattr(paper, 'abstract'):
```

Comment:
lines 55-66 are overkill; just use ``getattr()`` and provide default values when you construct the fixture, below